src/couch_replicator_tasks.erl

```diff
@@ -0,0 +1,85 @@
+-module(couch_replicator_tasks).
```

Comment:
Why is this a gen_server? It seems to be just a function that pretty-prints the ETS table state.